easybuild/easyconfigs/p/plc/plc-2.0-intel-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,86 @@
+easyblock = 'Waf'
+
+name = 'plc'
+version = '2.0'
+
+homepage = 'http://pla.esac.esa.int/pla/#home'
+
+description = """
+plc is the public Planck Likelihood Code.
+It provides C and Fortran libraries that allow 
+users to compute the log likelihoods of the temperature, 
+polarization, and lensing maps.  Optionally, it also provides a python version of this library, 
+as well as tools to modify the predetermined options for some likelihoods 
+(e.g. changing the high-ell and low-ell lmin and lmax values of the temperature).
+"""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = ['http://pla.esac.esa.int/pla/aio']
+sources = ['product-action?COSMOLOGY.FILE_ID=COM_Likelihood_Code-v2.0_R2.00.tar.bz2']
+
+patches = [
+    'plc-2.0-fix-build.patch',		 # workarounds fixing waf build behavior in EB environment
+    'plc-2.0-fix-openmp-flags.patch',	 # add proper (-fopenmp) openmp flags for icc and ifort
+]
+
+checksums = [
+    # product-action?COSMOLOGY.FILE_ID=COM_Likelihood_Code-v2.0_R2.00.tar.bz2:
+    'cab9917723b0eb69e7b2521bb074aaabb412aac6eb36e902cd9372db67ac328e',
+    # plc-2.0-fix-build.patch:
+    'fdbc8e221e5c49f6df70c6e9505ca9f25332ffe2483633999e524ecddb4fe67c',
+    # plc-2.0-fix-openmp-flags.patch:
+    '99a4c310b37f1a3d597d7a47b2f2ac714b12d7bcb00b8ad6a97464f0a454428c',
+]
+
+pyver_maj_min = '2.7'
+pyver = '%s.15' % pyver_maj_min
```

Comment:
@jose-d Please look into using `%(pyver)s` and `%(pyshortver)s` templates, see any recent easyconfig that relies on Python...

Reply:
new easyconfig written with _more recent_ easyconfig code-style.